pm_uep_plot/pm_uep_plot.xml

```diff
@@ -0,0 +1,82 @@
+<tool id="pm_uep_plot" name="generate plots from charge density" version="0.1.1" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">pymuonsuite</requirement>
+        <requirement type="package">zip</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s $charge_density $charge_density.name &&
+        x="$charge_density.name" &&
+        echo "calculated seed name \$x" &&
+        ln -s $castep_log "\${x%%.*}.castep" &&
+        touch "\${x%%.*}.yaml" &&
+        printf "chden_path: ./ \n" >>"\${x%%.*}.yaml" &&
+        ([[ ! -z "$chden_seed" ]] && printf "chden_seed: $chden_seed \n">>"\${x%%.*}.yaml" || ( >&2 echo "chden_seed empty")) &&
+        ([[ ! -z "$gw_factor" ]] && printf "gw_factor: $gw_factor \n">>"\${x%%.*}.yaml" || ( >&2 echo "gw_factor empty")) &&
+        ([[ ! -z "$line_plots" ]] && ( printf "line_plots: $line_plots \n" | sed "s/__ob__/[/g" | sed "s/__cb__/]/g" )>>"\${x%%.*}.yaml" || ( >&2 echo "line_plots empty" && exit 2)) &&
+        ([[ ! -z "$plane_plots" ]] && ( printf "plane_plots: $plane_plots \n" | sed "s/__ob__/[/g" | sed "s/__cb__/]/g" )>>"\${x%%.*}.yaml" || ( >&2 echo "plane_plots empty" && exit 2)) &&
+        cat "\${x%%.*}.yaml" &&
+        pm-uep-plot "\${x%%.*}.yaml" &&
+        ls > tree.txt &&
+        zip out.zip * -x "tree.txt"
```

Comment:
Integration test: this zip is picking up a `conda_activate.log` file when I run it. Probably better to explicitly include the things you want to zip (I don't know what chaos might be caused on other instances by this)